test/auth/sasl-imap-smtp-auth.txt

```diff
@@ -0,0 +1,5 @@
+EHLO mail
```

Comment:
Perhaps you should reuse the smpt auth (renaming the file to something more obvious, specifying the user perhaps). Because if we change a password, it should be done in both file (the probably look the same)